packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2146,6 +2148,7 @@
   "Require_password_change": "Require password change",
   "Resend_verification_email": "Resend verification email",
   "Reset": "Reset",
+  "Reset_now": "Reset now",
```

Comment:
Please don't add a new key, use the one above.

Reply:
@thiagosanchz what do you think about change that texts(remove the `now`)? I really like the now words :/

Reply:
@engelgabriel @ggazzo Why we can't add new words? I think the word NOW emphasizes the action.

Reply:
@thiagosanchz @ggazzo @brunosquadros @sampaiodiego let's not create new translation keys like this, that are important and don't add much to the user experience. Every new KEY will have to be translated into 50 languages and maintained. We already have far more keys that we need and we have a big mess because of it. We are trying to reduce the number of redundant translation keys.